### ISSUE TITLE
No Calendar were shown on the right screen of Calendar APP 2.1.2

### ISSUE THREAD

**Issue by wxf26054** [nextcloud/calendar#2734]
There were no calendar table on the right of the Calendar App
![image](https://user-images.githubusercontent.com/22016320/101593044-45dbc400-3a2a-11eb-8ac8-27fb86961847.png)

**Comment by fweng322** [comment]
I have the same problem after upgrading Nextcloud to 20.0.3.

There are errors in F12 console: https://pastebin.com/yLekDmTg

**Comment by wxf26054** [comment]
This is my error log in F12 Console. 
http://cloud.amazehome.xyz/index.php/s/BkEQkNQiX79BmWK

**Comment by fweng322** [comment]
Same after upgrading to 20.0.4.

**Comment by Quix0r** [comment]
Also possible duplicate #2775 and all point to this error: `ReferenceError: ResizeObserver is not defined`

**Comment by PatriceVigier** [comment]
Calendard 2.1.3
Nextcloud 20.0.5.2
White page on Firefox 69.0.3
The console error is
TypeError: "e.replaceAll is not a function"
and
TypeError: "this.$refs.fullCalendar is undefined"

Work on Chrome 87

**Comment by The-Compiler** [comment]
Likely fixed by #2822?

**Comment by rysiekpl** [comment]
It's been a month, any chance this will get resolved in a release anytime soon? For some of us switching a browser is not an option, and Firefox is still one of the main browsers out there.

**Comment by PatriceVigier** [comment]
and Firefox is a browser that respect your private life, what Chrome or MS do not, this is why it is important to support them

**Comment by fweng322** [comment]
> It's been a month, any chance this will get resolved in a release anytime soon? For some of us switching a browser is not an option, and Firefox is still one of the main browsers out there.

In my instance (https://nextcloud.slat.org) it has been fixed, also worked on Firefox. 

![Screenshot_20210220_221852](https://user-images.githubusercontent.com/1000315/108598639-b22f5800-73c9-11eb-956e-4bba3c4cc262.png)

version calendar 2.1.3, nextcloud 20.0.7.

**Comment by tcitworld** [comment]
> It's been a month, any chance this will get resolved in a release anytime soon? For some of us switching a browser is not an option, and Firefox is still one of the main browsers out there.

Firefox versions which are not 3 years old work perfectly, that is not the topic.

**Comment by rysiekpl** [comment]
> Firefox versions which are not 3 years old work perfectly, that is not the topic.

I'm on Firefox 73.0.1, [it is a year old](https://en.wikipedia.org/wiki/Firefox_version_history#Firefox_68_through_77):
> Firefox 73 was released on February 11, 2020 for desktop only.

Even if the problem is the browser version, then how about communicating this clearly by closing this issue and making a clear comment about it, instead of being passive-aggressive about "3-year-old" browsers without even asking me what version is involved? My version is newer than the one mentioned before in this thread; are you saying I am responsible for testing every single Firefox version out there to try to figure out which version works?

Any of the above (asking me about the Ff version, or closing the issue with clear info which lowest version of Ff is officially supported) could have put me on the right path. Instead, you just made me less inclined to recommend Nextcloud to anyone in the future. I have managed 4 different Nextcloud instances so far, including for a rather large org, moving some of them from ownCloud specifically due to how bad the community around it was. Thanks for this splendid new experience.

For the record, here's Calendar 2.1.3 on Nextcloud 20.0.7 under Firefox 73.0.1.

![Screenshot_2021-02-20 February 2021 - Calendar - Cloud](https://user-images.githubusercontent.com/6271171/108599736-e7688580-738a-11eb-9c23-e4dfbf977a9a.png)

**Comment by tcitworld** [comment]
Sorry about that, I mixed the two issues at hand here. My point was that switching browsers is not our answer, and we *really* don't deliberately break support for Firefox, only for old versions.

The `replaceAll` issue was fixed in https://github.com/nextcloud/calendar/pull/2822 and the ResizeObserver one in #2620 (this one is available in v2.1.3). I'll close this issue for now then, and I've asked for a release soon.
